guides/stargaze-studio/interact-with-a-collection/execute-messages.md

```diff
@@ -42,33 +80,27 @@ Takes an address and a token ID, then transfers that token to the address.
 
 * **Batch Transfer Tokens**
 
-Takes an address and multiple token IDs (separated by commas), then transfers all of the tokens to the address.
-
-{% hint style="info" %}
-The token IDs must be separated by commas:
-
-e.g., 2, 4, 8.
-
-Alternatively, you can give a range of IDs separated by a colon:
-
-e.g., 8:13
-{% endhint %}
+Takes an address and multiple token IDs (separated by commas), then transfers all of the tokens to the address. The format for listing token IDs is the same as **Batch Mint For** operation.
 
 * **Burn Token**
 
 Burns the token with the given ID.
 
 * **Batch Burn Tokens**
 
-Takes multiple token IDs and burns them at once. For the input format, check out the information box under the **Batch Transfer Tokens**.
+Takes multiple token IDs and burns them at once. The format for listing token IDs is the same as **Batch Mint For** operation.
 
 {% hint style="danger" %}
 Burn actions are irreversible. The burned tokens will never be accessible again.
 {% endhint %}
 
 * **Shuffle the token IDs**
 
-Shuffles all of the token IDs inside the collection.&#x20;
+Shuffles all of the token IDs inside the collection. Collections are created in the same order as they are sent by the user, by default. Executing this message will shuffle all the IDs to make the orders random.
+
+{% hint style="info" %}
+The shuffling price is 500 $STARS.
+{% endhint %}
 
 * **Airdrop Tokens**
```

Comment:
Airdrop Tokens now accepts a .csv file, which includes the header "address,amount", to facilitate airdropping multiple tokens to individual addresses.